x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -273,12 +275,31 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
                       {alert.executionStatus.error?.message}
                     </EuiText>
                     <EuiSpacer size="s" />
-                    <EuiButton color="danger" onClick={() => setDissmissAlertErrors(true)}>
-                      <FormattedMessage
-                        id="xpack.triggersActionsUI.sections.alertDetails.dismissButtonTitle"
-                        defaultMessage="Dismiss"
-                      />
-                    </EuiButton>
+                    <EuiFlexGroup gutterSize="s" wrap={true}>
+                      <EuiFlexItem grow={false}>
+                        <EuiButton color="danger" onClick={() => setDissmissAlertErrors(true)}>
+                          <FormattedMessage
+                            id="xpack.triggersActionsUI.sections.alertDetails.dismissButtonTitle"
+                            defaultMessage="Dismiss"
+                          />
+                        </EuiButton>
+                      </EuiFlexItem>
+                      {alert.executionStatus.error?.reason ===
+                        AlertExecutionStatusErrorReasons.License && (
+                        <EuiFlexItem grow={false}>
+                          <EuiButton
```

Comment:
I think this should be an `EuiButtonEmpty` (with `color=danger` and no icon).  So we can avoid conflicting button colors.

Reply:
<img width="1217" alt="Screen Shot 2020-12-10 at 11 43 52 AM" src="https://user-images.githubusercontent.com/55110838/101821581-ff936d00-3adc-11eb-991b-a9d735388227.png">

Reply:
👍 Thanks!